docs/scenarios/manifests/README.md

```diff
@@ -0,0 +1,65 @@
+## Generating manifests using CPMA and applying them
+
+CPMA analyzes OCP3 configuration files and when possible transforms them to CRs, generated CRs are used to configure OCP4 cluster as `day two` operations.
+
+---
+
+
+### 1. Prerequisites
+
+Prior to working with CPMA you need to deploy an OCP 3.7+ cluster(OCP 3.7, 3.9, 3.10, 3.11 will be supported).
+There are to 2 ways for cpma to obtain desired configurations:
+
+1. SSH, CPMA is capable to connect to cluster using SSH and fetch configurations to a local directory. For using SSH it's required to have a proper private key and SSH user may require sudo permissions for reading configurations.
+
+2. Configurations stored in local directory, it's possible to manualy download configuration and feed them to CPMA.
+
+---
+
+### 2. General CPMA configuration
+
+CPMA can be configured using either:
+
+1. Interactive prompts. This is simillar to `openshift-install`. The tool can be run with no configuration, all missing values will be prompted. You can see an example below. This is the most recommended way, because prompts will guide you through needed values and it can generate a configuration based on prompted values that will be used later.
```

Comment:
Maybe clearer to say:
s/all missing values/all required values/

Reply:
Also I'd mention that the prompts allows to save the configuration which can be re-used at will.